src/components/Text.tsx

```diff
@@ -67,6 +69,21 @@ export const Text = styled.div<TextProps>`
         css`
             color: ${color};
         `}
+
+    ${({ ellipsis }) =>
+        ellipsis &&
+        css`
+            text-overflow: ellipsis;
+            overflow: hidden;
+        `}
+
+    ${({ lines }) =>
```

Comment:
Maybe here needs add additional check that `ellipsis is true`?

Reply:
done